docker-compose.yml

```diff
@@ -4,7 +4,7 @@ dnsdock:
         - "172.17.42.1:53:53/udp"
     volumes: 
         - /var/run/docker.sock:/var/run/docker.sock
-        - /Users/kmadel/dev/CloudBees/docker/cbdemo:/data
+        - /Users/David/Projects/CloudBees/docker/cbdemo:/data
```

Comment:
We need to figure out a strategy around user specific paths - could see if ~ works, and then align on a standard directory structure: `~/projects/cloudbees/docker/cbdemo`